receiver/vcenterreceiver/receiver.go

```diff
@@ -0,0 +1,56 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vcenterreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/vcenterreceiver"
+
+import (
+	"context"
+	"fmt"
+
+	"go.opentelemetry.io/collector/component"
+	"go.uber.org/multierr"
+	"go.uber.org/zap"
+)
+
+var _ component.Receiver = (*vcenterReceiver)(nil)
+
+type vcenterReceiver struct {
+	config  *Config
+	logger  *zap.Logger
+	scraper component.Receiver
+}
+
+func (v *vcenterReceiver) Start(ctx context.Context, host component.Host) error {
+	var err error
+	if v.scraper != nil && v.config.MetricsConfig.Endpoint != "" {
```

Comment:
If the endpoint is nil, should this be running?

Reply:
The endpoint is defined as a `string` type so I don't think it can be defined as nil right? Wouldn't the zero value just be an empty string? 

Mostly this is here if they wanted to use this component for just the follow up logs pipeline and don't care about the metrics pipeline. In this case would mean that the Endpoint would be an empty string and we wouldn't start the metric scraper component.

Reply:
Makes sense. Thanks.